src/components/investigators/DeleteInvestigator.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Button, Card } from 'react-bootstrap';
+import Swal from 'sweetalert2';
+import { deleteInvestigator } from '../../redux/DeleteInvestigator/deleteInvestigatorSlice';
+import { fetchInvestigators } from '../../redux/investigators/investigatorSlice';
+
+const DeleteInvestigator = () => {
```

Comment:
- [ ] When I try to delete an investigator, this happens:


![image](https://user-images.githubusercontent.com/57421823/227433949-9ae2fc96-6582-4429-b0c2-af834e53d213.png)

The app is claiming I'm not logged in.

If this is a bug, kindly fix it so your app runs as intended. 👍